src/centrality.c

```diff
@@ -1971,7 +1991,9 @@ int igraph_betweenness_estimate(const igraph_t *graph, igraph_vector_t *res,
                     distance[neighbor] = distance[actnode] + 1;
                     IGRAPH_CHECK(igraph_dqueue_push(&q, neighbor));
                 }
-                if (distance[neighbor] == distance[actnode] + 1) {
+                if (distance[neighbor] == distance[actnode] + 1 &&
+                    (distance[neighbor] <= cutoff + 1 || cutoff < 0)) {
```

Comment:
Hi Vincent, sorry this is what I meant with the zero cutoff. Most places in the code divide into > 0 and <=0, but here it's different, just double checking this is what you meant.

Reply:
Ah, yes, good catch, thanks!